### DISCUSSION TITLE
Action Blocked Error

### DISCUSSION THREAD

**Discussion by antedap** [SFX-WoW/Masque#224] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.1.5.41031

### Add-On Version

9.1.1

### Description

I think the problem is a conflict with World Quest Tracker and Masque, when I try to list a group for world quest the add-on should compile automatically the title of the group, but Masque is blocking it.

### Steps To Reproduce

1. Using World Quest Tracker
2. Click on the Find group button
3. Start group in the dungeon finder

### Error Message

```
2x [ADDON_ACTION_BLOCKED] AddOn 'Masque' tried to call the protected function 'CreateListing()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `CreateListing'
[string "@FrameXML\LFGList.lua"]:1025: in function `LFGListEntryCreation_ListGroupInternal'
[string "@FrameXML\LFGList.lua"]:1085: in function `LFGListEntryCreation_CheckAutoCreate'
[string "@FrameXML\LFGList.lua"]:656: in function `LFGListEntryCreation_Show'
[string "@FrameXML\LFGList.lua"]:2057: in function <FrameXML\LFGList.lua:2054>
[string "=[C]"]: ?
[string "=[C]"]: ?
```

### Screenshots

http://prntscr.com/200u9uc

**Comment by StormFX** [comment, marked as answer]
This is an error propagated to Masque by something else (probably via a library). Masque doesn't call that function.

**Comment by antedap** [comment]
oh, thank you for the fast reply!
